rr.latex

```diff
@@ -67,7 +66,7 @@ With the basic functionality in place, we discovered that the main performance b
 our applications was the
 context switching required by using {\tt ptrace} to monitor system calls. We implemented
 a novel {\it system-call buffering} optimization to eliminate those context switches, dramatically
-reducing recording and replay overhead. This optimization relies on modern Linux kernel
+reducing recording and replay overhead on important real-world workloads. This optimization relies on modern Linux kernel
```

Comment:
"dramatically" on its own feels a little subjective and maybe hard to defend, but I think the results in the table are pretty dramatic so sticking with that :).  Also re-emphasizing that we're not talking about toy programs.